packages/firestore/src/remote/datastore.ts

```diff
@@ -158,3 +91,79 @@ export class Datastore {
       });
   }
 }
+
+export function invokeCommitRpc(
+  datastore: Datastore,
+  mutations: Mutation[]
+): Promise<MutationResult[]> {
+  const params: CommitRequest = {
+    database: datastore.serializer.encodedDatabaseId,
+    writes: mutations.map(m => datastore.serializer.toMutation(m))
+  };
+  return datastore
+    .invokeRPC<CommitRequest, api.CommitResponse>('Commit', params)
+    .then(response => {
+      return datastore.serializer.fromWriteResults(
+        response.writeResults,
+        response.commitTime
+      );
+    });
+}
+
+export function invokeBatchGetDocumentsRpc(
+  datastore: Datastore,
+  keys: DocumentKey[]
+): Promise<MaybeDocument[]> {
+  const params: BatchGetDocumentsRequest = {
+    database: datastore.serializer.encodedDatabaseId,
+    documents: keys.map(k => datastore.serializer.toName(k))
+  };
+
+  return datastore
+    .invokeStreamingRPC<
+      BatchGetDocumentsRequest,
+      api.BatchGetDocumentsResponse
```

Comment:
`BatchGetDocumentsRequest` is `api. BatchGetDocumentsRequest` with the `database` field set. I just added it in the Proto d.ts field since I/we have been editing this file manually lately.